client/my-sites/stats/site.jsx

```diff
@@ -132,6 +143,9 @@ class StatsSite extends Component {
 				/>
 				<div id="my-stats-content">
 					{ config.isEnabled( 'onboarding-checklist' ) && <ChecklistBanner siteId={ siteId } /> }
+					{ config.isEnabled( 'google-my-business' ) &&
```

Comment:
What you think about having that as a ([memoized](https://github.com/Automattic/wp-calypso/tree/master/client/lib/create-selector)) selector?

```js
const shouldShowGMB = ( state, siteId ) => {
   const siteGoals = ( getSiteOption( state, siteId, 'site_goals' ) || '' ).split( ',' );
   const sitePlan = getSitePlanSlug( state, siteId );
   return siteGoals.indexOf( 'promote' ) !== -1 &&
        [ PLAN_PREMIUM, PLAN_BUSINESS ].indexOf( sitePlan ) !== -1;
};
```